hackathon_site/requirements.txt

```diff
@@ -22,6 +22,7 @@ Jinja2==2.11.2
 MarkupSafe==1.1.1
 packaging==20.4
 pathspec==0.8.0
+Pillow==7.2.0
```

Comment:
What's this for?

Reply:
It's used by `ImageField` to figure out if the file is an image or not. It's not part of Django's dependencies I assume since most people don't need it.